spec/forms/steps/case/charges_form_spec.rb

```diff
@@ -53,39 +60,42 @@
         let(:application) { CrimeApplication.create }
         let(:case_record) { Case.create(crime_application: application) }
 
-        let(:offence_dates) { [
-          OffenceDate.new(date: Date.new(2000,11,03)),
-          OffenceDate.new(date: Date.new(2009,05,01))
-        ] }
+        let(:offence_dates) do
+          [
+            OffenceDate.new(date: Date.new(2000, 11, 0o3)),
+            OffenceDate.new(date: Date.new(2009, 0o5, 0o1))
```

Comment:
Not sure I understand what these "o"s are about?

Reply:
The leading zero seems confuse rubocop. I'll remove the leading zero for now rather than converting to base 8!